compiler/src/dmd/declaration.d

```diff
@@ -244,6 +244,7 @@ extern (C++) abstract class Declaration : Dsymbol
       enum wasRead    = 1; // set if AliasDeclaration was read
       enum ignoreRead = 2; // ignore any reads of AliasDeclaration
       enum nounderscore = 4; // don't prepend _ to mangled name
+      enum hidden       = 8; // don't print this in .di files
```

Comment:
also need to update the corresponding header.

Reply:
No they don't, this is a compile time constant.